user/imodule.go

```diff
@@ -245,8 +247,12 @@ func (this *Module) Decode(em *ebpf.Map, b []byte) (result string, err error) {
 	}
 	return
 }
-
+func (this *Module) GetConf() IConfig {
+	return this.conf
```

Comment:
unnsed code.

Reply:
it was used on [user/event_bash.go](https://github.com/ehids/ecapture/pull/53/files/8cba25a9812fc4e98fb4a9014f13ea71a4e402dd#diff-b6607bfa6c5c99ecb9aa770cce82ba3465ab15455a0d7bec77be93336ea18cd3)
```
errno :=  GetModuleByName(MODULE_NAME_BASH).GetConf().(*BashConfig).ErrNo
```
is there any other ways to get conf from Imodule if don't use this func?

Reply:
```go
errno :=  this.Module().GetConf().(*BashConfig).ErrNo
```